relay-general/src/processor/attrs.rs

```diff
@@ -717,6 +717,7 @@ impl<'a> Path<'a> {
             SelectorSpec::And(ref xs) => xs.iter().all(|x| self.matches_selector(x)),
             SelectorSpec::Or(ref xs) => xs.iter().any(|x| self.matches_selector(x)),
             SelectorSpec::Not(ref x) => !self.matches_selector(x),
+            SelectorSpec::Other(_) => false,
```

Comment:
I'm a bit unsure on this one, also, should I add the relay_warn log here, in the way that olek did here: https://github.com/getsentry/relay/pull/1859/files

Reply:
I think this is a good idea!